libdeno/BUILD.gn

```diff
@@ -16,21 +13,27 @@ config("deno_config") {
   }
 }
 
-v8_static_library("v8") {
-  public_deps = [
-    "//build/win:default_exe_manifest",
+v8_component("v8") {
```

Comment:
I guess this breaks the prebuild V8. Maybe delete the prebuilt files until we can bring them back?